skymap_scanner/server/start_scan.py

```diff
@@ -695,17 +755,20 @@ async def serve_scan_iteration(
 
     # get pixels & send to client(s)
     LOGGER.info("Getting pixels to send to clients...")
+    pixreco_ids_sent = set([])
     async with to_clients_queue.open_pub() as pub:
         for i, pframe in enumerate(pixeler.generate_pframes()):  # queue_to_clients
+            _tup = pixelreco.pixel_to_tuple(pframe)
             LOGGER.info(
-                f"Sending message M#{i} ({pixelreco.pixel_to_tuple(pframe)})..."
+                f"Sending message M#{i} ({_tup})..."
             )
             await pub.send(
                 {
                     cfg.MSG_KEY_RECO_ALGO: reco_algo,
                     cfg.MSG_KEY_PFRAME: pframe,
                 }
             )
+            pixreco_ids_sent.add(_tup)
```

Comment:
Following my comment above -- with `pixreco_ids_sent`, I'm not sure there's a need to have `n_pixreco`. We can replace all mentions of  `n_pixreco` with  `pixreco_ids_sent`, essentially all the way until the `ProgressReporter` instantiation.

Reply:
I removed the argument from the `PixelRecoCollector`, which I think is what you were suggesting?

Reply:
Looks great!